src/boot/efi/boot.c

```diff
@@ -521,7 +521,7 @@ static BOOLEAN menu_run(
         uefi_call_wrapper(ST->ConOut->SetAttribute, 2, ST->ConOut, EFI_LIGHTGRAY|EFI_BACKGROUND_BLACK);
 
         /* draw a single character to make ClearScreen work on some firmware */
-        uefi_call_wrapper(ST->ConOut->OutputString, 2, ST->ConOut, L" ");
+        uefi_call_wrapper(ST->ConOut->OutputString, 2, ST->ConOut, (CHAR16*) L" ");
```

Comment:
why drop the qualifiers here? shouldn't matter, no?

Reply:
`OutputString()` takes `CHAR16 *` instead of `const CHAR16 *`. So, it is necessary to cast the string, unfortunately.